packages/notifi-dapp-example/src/components/DiscordInput.tsx

```diff
@@ -1,16 +1,27 @@
 import { Icon } from '@/assets/Icon';
 import { useNotifiTargets } from '@/hooks/useNotifiTargets';
-import { useNotifiSubscriptionContext } from '@notifi-network/notifi-react-card';
+import {
+  useNotifiForm,
+  useNotifiSubscriptionContext,
+} from '@notifi-network/notifi-react-card';
 import React from 'react';
 
 import { Toggle } from './Toggle';
 
 export type DiscordInputProps = Readonly<{
   disabled: boolean;
+  isEdit?: boolean;
 }>;
 
-export const DiscordInput: React.FC<DiscordInputProps> = ({ disabled }) => {
-  const { useDiscord } = useNotifiSubscriptionContext();
+export const DiscordInput: React.FC<DiscordInputProps> = ({
+  disabled,
+  isEdit,
```

Comment:
Just a small nit, the `isEdit` is kind of confusing for me. Not sure if `isEditable` better?